doc/source/conf.py

```diff
@@ -82,6 +81,7 @@
     "sphinx_gallery.gen_gallery",
     "sphinxemoji.sphinxemoji",
     "sphinx.ext.graphviz",
+    "sphinx_reredirects",
```

Comment:
Hey.. this is awesome, didn't even know it existed. You should present it in the next meeting!

Reply:
Ofc.... Count with it...

![](https://media.giphy.com/media/YrMrSUfeh5do2FISt8/giphy.gif)

Reply:
It is still a beta thing in PyMAPDL, let me be sure it works how I think it works. But surely.